src/AWS.Deploy.Common/Recipes/OptionSettingItem.ValueOverride.cs

```diff
@@ -85,8 +89,22 @@ public object GetValue(IDictionary<string, string> replacementTokens, IDictionar
             return DefaultValue;
         }
 
+        /// <summary>
+        /// Assigns this Item a new value.
+        /// </summary>
+        /// <exception cref="ValidationFailedException">
+        /// Thrown if one or more <see cref="Validators"/> determine
+        /// <paramref name="valueOverride"/> is not valid.
+        /// </exception>
         public void SetValueOverride(object valueOverride)
         {
+            foreach (var validator in this.BuildValidators())
```

Comment:
If a user wants to proceed with the default values of any `OptionSettingItem`, then the validator for it will be unused and storing it as part of the `OptionSettingItem` will not serve any purpose. Hence, IMO its better to build the validators on the fly and ONLY for those `OptionSettingItem` which the user wants to customize.